14/umbraco-cms/reference/management-api/README.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Get started with the Management API.
+---
+
+# Management API
+
+The Management API delivers headless capabilities built directly into Umbraco. Unlike the Content Delivery API, the Management API enables interaction with everything else in the backoffice except for the Content and Media items.
```

Comment:
> everything else in the backoffice except for the Content and Media items.

This is not true. Content and media items are indeed also managed with the Management API. I think the sentence should rather be something along the lines of "the Management API allows for managing everything within the backoffice".